webui/react/src/shared/contexts/UIStore.ts

```diff
@@ -0,0 +1,73 @@
+import { DarkLight, Mode, Theme } from '../themes';
+
+export interface StateUI {
+  chromeCollapsed: boolean;
+  darkLight: DarkLight;
+  isPageHidden: boolean;
+  mode: Mode;
+  showChrome: boolean;
+  showSpinner: boolean;
+  theme: Theme;
+}
+
+export const initUI: StateUI = {
+  chromeCollapsed: false,
+  darkLight: DarkLight.Light,
+  isPageHidden: false,
+  mode: Mode.System,
+  showChrome: true,
+  showSpinner: false,
+  theme: {} as Theme,
+};
+
+export enum StoreActionUI {
```

Comment:
this is consistent with how we have these in Store but maybe we want a shorter name here